packages/core/src/v3/schemas/common.ts

```diff
@@ -49,6 +50,7 @@ export const TaskRunInternalError = z.object({
     "TASK_RUN_CANCELLED",
     "TASK_OUTPUT_ERROR",
     "HANDLE_ERROR_ERROR",
+    "GRACEFUL_EXIT_TIMEOUT"
```

Comment:
What happens if someone is on an older version of the SDK and receives one of these new errors from the platform? Or isn't that possible?

Reply:
Those errors get sent from the worker to the platform, so that wouldn't be possible.